deployment/terraform/firewall.tf

```diff
@@ -332,10 +332,20 @@ resource "aws_security_group_rule" "app_logstash_egress" {
   source_security_group_id = module.database_enc.database_security_group_id
 }
 
+resource "aws_security_group_rule" "app_logstash_opensearch_egress" {
+  type      = "egress"
+  from_port = var.logstash_port
+  to_port   = var.logstash_port
+  protocol  = "tcp"
+
+  security_group_id        = aws_security_group.app_logstash.id
+  source_security_group_id = aws_security_group.opensearch.id
+}
+
 resource "aws_security_group_rule" "app_logstash_https_egress" {
   type             = "egress"
-  from_port        = 443
-  to_port          = 443
+  from_port        = var.logstash_port
+  to_port          = var.logstash_port
```

Comment:
This egress rule was added for ECR to fetch the Docker image. I think it is just a coincidence that we need the same port for OpenSearch. Let's revert it if it is not required?